prisma/postgres/schema.prisma

```diff
@@ -3,7 +3,7 @@
 
 generator client {
     provider = "prisma-client-js"
-    output = "../../node_modules/@prisma-db-psql/client"
```

Comment:
Extra space?

Reply:
haha that was the default then i removed it then i added it back? idk yea shud remove but doesnt affect behaviour for now

Reply:
oh is from formatter ah